### ISSUE TITLE
No icon for the world's tallest Ferris wheel

### ISSUE THREAD

**Issue by jidanni** [gravitystorm/openstreetmap-carto#3550]
Taiwan's tallest Ferris wheel is indicated by just a child's see-saw.
https://www.openstreetmap.org/node/5513773274

Taipei's Ferris wheel isn't even indicated,
https://www.openstreetmap.org/node/5121602758

No icon for the world's tallest Ferris wheel either,
https://www.openstreetmap.org/node/3521729497

**Comment by Adamant36** [comment]
There doesn't seem to be anything on the tourism=attraction wiki page for the attraction=* tag and it didn't come up in search. Do you know its status or have any other information on it? It looks like it has 23,207 uses. So it might be worth rendering icons for, but it would probably at least need a wiki page first. 

Currently issue #3545 is discussing removing name rendering from things tagged as tourism=attraction. That might be an issue in light of this or at least worth considering. I think you'll be in an even worse place if the ferris wheel doesn't even have its name rendered anymore. So it might be worth sharing your opinion there. 

outside of that, I could have swore there was another tag besides attraction=* for things found in theme parks, but I can't seem to find it. So maybe I'm wrong.

**Comment by jidanni** [comment]
All I know is we see something like
https://wiki.openstreetmap.org/wiki/File:Bigwheel.png
in the (iD) editor, but then it doesn't make it into the final product.

I mean these things can be seen several counties away, which should earn
them at least an icon.

**Comment by jeisenbe** [comment]
Attraction=* is not actually a subkey for tourism=attraction, rather it is
meant to tag various rides, animals etc in Theme Parks, Zoos and similar
places.

Since most of these are smaller features within a theme park or zoo,
normally they would not be rendered very prominently. I wonder if there is
a better tag for the huge sight-seeing Ferris wheels in big cities?

**Comment by Adamant36** [comment]
Makes sense. I agree with you that there could be a better tag. As far as I knew huge ferris wheels like the one London aren't actually in themes parks anyway.

**Comment by kocio-pl** [comment]
Looks like sane object to render, given estimated low number of them and relatively big importance, we have enough of them already in my opinion (318) and the numbers are growing fast, it is also documented:

https://taginfo.openstreetmap.org/tags/attraction=big_wheel
https://wiki.openstreetmap.org/wiki/Key:attraction

![taghistory 31](https://user-images.githubusercontent.com/5439713/49580940-922e2e00-f950-11e8-8fa7-92544e01c813.png)

**Comment by Tomasz-W** [comment]
Icon proposal:

* ![attraction big_wheel](https://user-images.githubusercontent.com/25656654/49601328-d2f46a00-f985-11e8-91c4-b05311584841.png) (v1)
* ![attraction big_wheel](https://user-images.githubusercontent.com/25656654/49602090-e4d70c80-f987-11e8-9928-78fd14e6bf7e.png) (v2)

Gist link: https://gist.github.com/Tomasz-W/033be1e9c86b6cf814039f250155655c

**Comment by kocio-pl** [comment]
What color should we use?

**Comment by polarbearing** [comment]
man-made was my first thought also, but it would be more within things like Zoo or Museum, thus amenity-brown is more logical.

**Comment by da1910** [comment]
I prefer the second logo from above, it's a bit bolder, how does it look in 14px size? Brown would seem to make more sense, at least for the big_wheel variety. I can take this if there's agreement?

**Comment by meased** [comment]
> how does it look in 14px size

@Tomasz-W 's icons are already 14x14.

As far as color, we do have a category for tourist attractions. Seems a more logical fit than amenity or man made. I do not believe we have ever had an attraction colored icon, but there's a first time for everything. Zoos have recently moved to this color.

**Comment by da1910** [comment]
Do we want an icon for every 'attraction=big_wheel', or just for those with 'tourism=attraction'? I'll post a couple of examples presently.

**Comment by da1910** [comment]
Here's some tests, currently drawing the icon at z14+ to match towers and the name at z17+, but moving that is easily done. Currently the liverpool wheel has two names as it's separately tagged tourism=attraction and as attraction=big_wheel.
![screenshot from 2019-02-15 16-32-44](https://user-images.githubusercontent.com/12086549/52870512-f420c600-313f-11e9-9d6a-c57c8614d2d1.png)
![screenshot from 2019-02-15 16-32-52](https://user-images.githubusercontent.com/12086549/52870516-f551f300-313f-11e9-8bbc-aedf3188cfa3.png)
![screenshot from 2019-02-15 16-34-29](https://user-images.githubusercontent.com/12086549/52870518-f71bb680-313f-11e9-8e1a-a427b7583275.png)

**Comment by jeisenbe** [comment]
> Do we want an icon for every 'attraction=big_wheel', or just for those
with 'tourism=attraction'?

The key “attraction” is not imported as a polygon, so it cannot yet be used
as a stand-alone key for the feature (though we could change this in >6
months when the database is next reloaded on the osm servers)

So for now we need to consider if “attraction=“ requires a
tourism=attraction tag, according to the wiki and usual practice in the
database.

**Comment by da1910** [comment]
The wiki entry for attraction=* claims that it is not a subkey of tourism=attraction

> This is no subkey of tourism=attraction since the attractions are usually part of bigger venture.

Tourism=attraction has 175k of which 11k have attraction=* entered, there are 25k total attraction=* tags.  I'm minded to render everything with attraction=big_wheel as above, unless there are reasonable objections?

**Comment by jeisenbe** [comment]
We can’t render any closed ways (areas) tagged “attraction=“ alone at this
time, so we will probably need to wait till the next database reload, when
this can be updated. This may be in 6 months to a year.

**Comment by da1910** [comment]
That's odd, as if that's the case then I shouldn't be seeing the icon in the liverpool example above. It's tagged as an attraction=big_wheel but no tourism tag and a separate building with tourism=attraction but no attraction tag. I don't know what's going on here in that case.
![screenshot from 2019-02-20 13-48-50](https://user-images.githubusercontent.com/12086549/53096201-7cbfad80-3516-11e9-8e07-32151f8481f0.png)
![screenshot from 2019-02-20 13-49-02](https://user-images.githubusercontent.com/12086549/53096202-7cbfad80-3516-11e9-8d64-3bbfa3511517.png)

**Comment by jeisenbe** [comment]
The feature on top with name=Wheelm of Liverpool is tagged
tourism=attraction; tourism tags are imported as polygons (eg tourism=hotel.

The second with name Echo Wheel if Liverpool is tagged building=yes, so
this will also be imported as a polygon.

**Comment by jidanni** [comment]
> This may be in 6 months to a year.

Now a year later commit da1910/openstreetmap-carto somehow is still not being rendered anywhere.

**Comment by HolgerJeromin** [comment]
> Now a year later commit da1910/openstreetmap-carto somehow is still not being rendered anywhere.

Again a very demanding tone from you :-(

Nitpick: 20.2.2020 is still in the future  :) 

But more important in a volonteer project is the work which has been doen: The next release will probably includes the required schema changes. Ref #4031

**Comment by jeisenbe** [comment]
@jidanni, the commit you are talking about (https://github.com/da1910/openstreetmap-carto/commit/01476ae2e4ab3340c89491a5a1f9352a0858f14b) is not in this repository, rather it was done by @da1910 in their own repository. 

So far there is no PR submitted to render this feature.

**Comment by da1910** [comment]
I can put in a pull request for the changes, but I don't know how likely it is that it will be approved, it will also likely need to be changed when the schema changes happen, so I suspect the first response will be to wait until then.
On the plus side, having several vocal supporters of the addition is a good sign!

**Comment by jeisenbe** [comment]
We did not end up including any `attraction=*` features in the schema changes, so any that are mapped as closed ways to represent areas would need to wait for the next database reload, including this feature.
